blobstore/list.go

```diff
@@ -66,20 +66,15 @@ func (bh *BlobHandler) HandleListByPrefix(c echo.Context) error {
 	}
 
 	delimiterParam := c.QueryParam("delimiter")
-	var delimiter bool
-	if delimiterParam == "true" || delimiterParam == "false" {
+	delimiter := true
+	if delimiterParam != "" {
 		delimiter, err = strconv.ParseBool(delimiterParam)
 		if err != nil {
 			errMsg := fmt.Errorf("error parsing `delimiter` param: %s", err.Error())
 			log.Error(errMsg.Error())
 			return c.JSON(http.StatusUnprocessableEntity, errMsg.Error())
 		}
 
-	} else {
-		errMsg := fmt.Errorf("request must include a `delimiter`, options are `true` or `false`")
-		log.Error(errMsg.Error())
-		return c.JSON(http.StatusUnprocessableEntity, errMsg.Error())
-
 	}
 	if delimiter && !strings.HasSuffix(prefix, "/") {
```

Comment:
Why dont you also need to do this in the detailed version of this function?

Reply:
It was because of the root directory check but I just added a tertiary check to see if the prefix is empty we don't add a '/' and added it for both of them. Thanks!